### DISCUSSION TITLE
VectorStore vs BasePydanticVectorStore

### DISCUSSION THREAD

**Discussion by zhenlohuang** [run-llama/llama_index#8698] [General]
In llama-index code, what is the difference between VectorStore and BasePydanticVectorStore? I found some of vector stores implemented VectorStore, but some of them implemented BasePydanticVectorStore.

**Comment by logan-markewich** [comment]
They are the same actually, the only difference is that the latter is serializable. Eventually, all vector stores will be based on pydantic :)
